app/views/membership_application_mailer/new_application.html.haml

```diff
@@ -0,0 +1,6 @@
+Hi #{@admin.first_name},
+
+%p A new application has been submitted by #{@membership_application.user_full_name}, with company number: #{@membership_application.company_number} and company id: #{@membership_application.company_id}.
+
+%br Regards,
+%em= ENV['SHF_ADMIN_EMAIL']
```

Comment:
@sic-f I think your best bet is to:

1) Wait for PR https://github.com/AgileVentures/shf-project/pull/318 to be merged (will be soon)
2) Implement PT https://www.pivotaltracker.com/story/show/149932678 (and have it merged)
3) Rebase this PR to develop, and then implement the requested changes.

Does that make sense?